ova/Libraries/provision-opendistro.sh

```diff
@@ -0,0 +1,306 @@
+
+        # Configure JVM options for Elasticsearch
+        ram_gb=$(free -g | awk '/^Mem:/{print $2}')
+        ram=$(( ${ram_gb} / 2 ))
+
+        if [ ${ram} -eq "0" ]; then
+            ram=1;
+        fi
+        eval "sed -i "s/-Xms1g/-Xms${ram}g/" /etc/elasticsearch/jvm.options $debug"
+        eval "sed -i "s/-Xmx1g/-Xmx${ram}g/" /etc/elasticsearch/jvm.options $debug"
+        jv=$(java -version 2>&1 | grep -o -m1 '1.8.0' )
+        if [ "$jv" == "1.8.0" ]
+        then
+            ln -s /usr/lib/jvm/java-1.8.0/lib/tools.jar /usr/share/elasticsearch/lib/
+            echo "root hard nproc 4096" >> /etc/security/limits.conf
+            echo "root soft nproc 4096" >> /etc/security/limits.conf
+            echo "elasticsearch hard nproc 4096" >> /etc/security/limits.conf
+            echo "elasticsearch soft nproc 4096" >> /etc/security/limits.conf
+            echo "bootstrap.system_call_filter: false" >> /etc/elasticsearch/elasticsearch.yml
+        fi
+
+        # Start Elasticsearch
+        startService "elasticsearch"
+        echo "Initializing Elasticsearch..."
+        until $(curl -XGET https://localhost:9200/ -uadmin:admin -k --max-time 120 --silent --output /dev/null); do
+            echo -ne $char
+            sleep 10
+        done
+
+        eval "cd /usr/share/elasticsearch/plugins/opendistro_security/tools/ $debug"
+        eval "./securityadmin.sh -cd ../securityconfig/ -nhnv -cacert /etc/elasticsearch/certs/root-ca.pem -cert /etc/elasticsearch/certs/admin.pem -key /etc/elasticsearch/certs/admin.key $debug"
+
+        echo "Done"
+    fi
+}
+
+## Filebeat
+installFilebeat() {
+    logger "Installing Filebeat..."
+
+    #eval "echo -e '[elasticsearch-7.x]\nname=Elasticsearch repository for 7.x packages\nbaseurl=https://artifacts.elastic.co/packages/7.x/yum\ngpgcheck=1\ngpgkey=https://artifacts.elastic.co/GPG-KEY-elasticsearch\nenabled=1\nautorefresh=1\ntype=rpm-md' | tee /etc/yum.repos.d/elastic.repo"
+
+    eval "$sys_type install filebeat-"${ELK_VERSION}" -y -q  $debug"
+    if [  "$?" != 0  ]
+    then
+        echo "Error: Filebeat installation failed"
+        exit 1;
+    else
+        eval "curl -so /etc/filebeat/filebeat.yml https://raw.githubusercontent.com/wazuh/wazuh/${BRANCH}/extensions/filebeat/7.x/filebeat_all_in_one.yml --max-time 300  $debug"
+        eval "curl -so /etc/filebeat/wazuh-template.json https://raw.githubusercontent.com/wazuh/wazuh/${BRANCH}/extensions/elasticsearch/7.x/wazuh-template.json --max-time 300 $debug"
+        eval "chmod go+r /etc/filebeat/wazuh-template.json $debug"
+        eval "curl -s https://packages.wazuh.com/3.x/filebeat/wazuh-filebeat-0.1.tar.gz --max-time 300 | tar -xvz -C /usr/share/filebeat/module $debug"
+        eval "mkdir /etc/filebeat/certs $debug"
+        eval "cp /etc/elasticsearch/certs/root-ca.pem /etc/filebeat/certs/ $debug"
+        ls mv /etc/elasticsearch/certs/
```

Comment:
This line will fail because `mv` file doesn't exist but, remove I would remove it anyway. It looks like this was added for debugging purposes.

Reply:
Done in 528feec208a367ba616fb87ea4237096d0681494